### ISSUE TITLE
"Stun" Length

### ISSUE THREAD

**Issue by Legacy-TacticalGamingInteractive** [LocalSpetsnaz/FiveM-Airsoft-Standalone#1]
Do you know if its possible for us to adjust this from the weapon side of things? Like you know how people have adjusted the stun gun script to have a shortly length of being stunned/downed. This is because of the heavy amount of abuse on these airsoft guns. People don't have a chance to get back up at all and just get sprayed forever by trolls. Perhaps if we can at least shorten the length of a time on the ground playing the stun animation we could mitigate this a bit.

Or, play a different animation when hit entirely.

**Comment by LocalSpetsnaz** [comment]
Yeah, I originally planned on having an animation play when hit where the player puts their hands up, but me being quite new to coding in general, I went with this. I'll update it when I find out how, If I have the time.
